.env.example

```diff
@@ -0,0 +1,33 @@
+# RPC
+RPC_URL=
```

Comment:
I think it would be great if you could add defaults here (where possible).

Reply:
f7385ecd335d742e23bfae6b0194d40c86b7dad5

Reply:
b67902df0b68edf20c5a8d9921487d3be7862029

Reply:
O boi! 

I won't overcomplicate this PR since is a bunch of nits. 

I will leave only APP_DATA and SETTLEMENT and a single RPC. I trust whomever clones this repo to set the right values :)